src/api/app/views/webui/user/user_profile_redesign/_info.html.haml

```diff
@@ -40,10 +40,19 @@
               = link_to(my_subscriptions_path, class: 'nav-link', title: 'Change Your Notifications') do
                 %i.fas.fa-lg.mr-2.fa-bell
                 %span.nav-item-name Change Your Notifications
+            %li.nav-item
+              = link_to(saved_replies_path, title: 'Saved replies', class: 'nav-link') do
+                %i.fas.fa-mail-bulk
```

Comment:
The action here looks a little different from the rest, just because some Bootstrap classes are missing: `fa-lg` and `mr-2`.
Simply add them taking the other list items as an example.

Reply:
Done. Thanks a lot for the review. 😄